staging/src/k8s.io/client-go/tools/leaderelection/leaderelection_test.go

```diff
@@ -208,83 +241,86 @@ func testTryAcquireOrRenew(t *testing.T, objectType string) {
 		},
 	}
 
-	for i, test := range tests {
-		// OnNewLeader is called async so we have to wait for it.
-		var wg sync.WaitGroup
-		wg.Add(1)
-		var reportedLeader string
-		var lock rl.Interface
+	for i := range tests {
+		test := &tests[i]
```

Comment:
Is this necessary? I tooks like a _, t reference won't change out from underneath the test.

Reply:
if it ever gets made parallel it would be - I can change if you like.